### PR TITLE
Creating bank-account exercise; tests and structure copied from java …

### PR DESCRIPTION
**Pull request by okaram**
…version

### CODE REVIEW

**Comment by m-a-ge** [comment]
@okaram can please add `Example.groovy`, it's required to run tests in our CI pipeline

Currently it files with the following:
```
Creating build for exercise 'BankAccount' in '/home/travis/build/exercism/groovy/exercises/bank-account' directory
Caught: java.nio.file.NoSuchFileException: /home/travis/build/exercism/groovy/exercises/bank-account/Example.groovy
java.nio.file.NoSuchFileException: /home/travis/build/exercism/groovy/exercises/bank-account/Example.groovy
```

**Comment by okaram** [comment]
Added

File: exercises/bank-account/BankAccountSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,202 @@
+@Grab('org.spockframework:spock-core:1.0-groovy-2.4')
+
+import spock.lang.Specification
+
+class TwoFerSpec extends Specification {
```
Comment (m-a-ge) [inline exercises/bank-account/BankAccountSpec.groovy]: Could please rename the class to `BankAccountSpec` for consistency

File: exercises/bank-account/BankAccountSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        
+        then:
+            bankAccount.getBalance() == 1000
+    }
+
+    void adjustBalanceConcurrently(BankAccount bankAccount) {
+        Random random = new Random();
+
+        Thread[] threads = new Thread[1000];
+        for (int i = 0; i < 1000; i++) {
```
Comment (m-a-ge) [inline exercises/bank-account/BankAccountSpec.groovy]: It's possible to make the code more Groovish by using constructions like closures etc:
````groovy
(1..1000).each { println it }
````

File: exercises/bank-account/BankAccountSpec.groovy [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+                        } catch (InterruptedException ignored) {
+                        } catch (Exception e) {
+                            fail("Exception should not be thrown: " + e.getMessage());
+                        } 
+                }
+            );
+            threads[i].start();
+        }
+
+        for (Thread thread : threads) {
```
Comment (m-a-ge) [inline exercises/bank-account/BankAccountSpec.groovy]: the same thing:
```groovy
l.each { println it }
```
These two comments are just for your considerations and it isn't mandatory to change anything, but what's the point of using Groovy just by copying Java code, right?

**Comment by m-a-ge** [comment]
@okaram I've merged your PR, thanks a lot for your contribution!

As a side note, please consider the following:
```groovy
List<Thread> threads = new ArrayList<Thread>(); // in Java
// is equivalent to
def threads = [] // in Groovy
```
The example above shows how simple you can write code in Groovy especially when types are not really required (in your case)

**Pull request merged**